qbraid/transpiler/conversions/qasm2/qasm2_to_cirq.py

```diff
@@ -44,7 +44,10 @@ def qasm2_to_cirq(qasm: Qasm2StringType) -> cirq.Circuit:
         Cirq circuit representation equivalent to the input QASM string.
     """
     try:
-        qasm = unfold_qasm2(qasm)
-        return cirq_qasm_parser.QasmParser().parse(qasm).circuit
+        qasm_module = pyqasm.loads(qasm)
+        qasm_module.unroll()
+        qasm_module.remove_barriers()  # cirq does not support barriers
```

Comment:
I know that again, this carries over from before. But here we should probably first check if the qasm program contains barriers, and if so, emit a warning telling the user that they are being removed bc cirq does not support barriers. Instead of having this done implicitly without the users knowledge

Reply:
https://github.com/qBraid/pyqasm/pull/85